docs/UsersGuide/source/BuildingRunningTesting/AQM.rst

```diff
@@ -0,0 +1,319 @@
+     - Concatenates the NEXUS emissions information into a single netCDF file (needed for the forecast) if NEXUS was split into multiple jobs using the ``NUM_SPLIT_NEXUS`` variable.
+   * - fire_emission
+     - Converts both satellite-retrieved gas and aerosol species emissions (RAVE) from mass (kg) to emissions rates (kg/m2/s) and creates 3-day hourly model-ready fire emissions input files.
+   * - point_source
+     - Aggregates the anthropogenic point source sectors of the National Emission Inventory (NEI) into a ready-to-input point-source emission file based on the weekday/weekend/holiday patterns of each sector and the date/time of the simulation.
+   * - aqm_ics
+     - Creates a chemical initial conditions file by using the previous cycle restart files. 
+   * - aqm_lbcs 
+     - Adds the chemical lateral boundary conditions (LBCs) to the meteorological LBCs to form the full set of ready-to-input LBCs for the simulation. This task includes two sub-tasks: (1) addition of the gaseous species LBCs and (2) addition of dynamic aerosol LBCs. The former adds static gaseous LBCs using monthly mean global data. The latter is the parallel job, which extracts the GEFS-Aerosol Model's output along the regional domain and performs the species conversion from :term:`GOCART` aerosols to CMAQ aerosols. 
+
+Post-processing Tasks of SRW-AQM
+------------------------------------
+
+The post-processing tasks for air quality modeling (AQM) are shown in :numref:`Table %s <TasksPostAQM>`. They are defined in the ``parm/wflow/aqm_post.yaml`` file. Since the module required to run these tasks is available only on WCOSS2, ``aqm_post.yaml`` should not be added to the ``rocoto: tasks: taskgroups:`` section of the configuration file ``config.yaml`` on other platforms.
+
+.. _TasksPostAQM:
+
+.. list-table:: Tasks for Post-processing of AQM
+   :widths: 20 50
+   :header-rows: 1
+
+   * - Task name
+     - Description
+   * - pre_post_stat
+     - Creates surface (i.e., model first level) meteorological and chemical files to support air quality product generation and generate training data to support bias correction tasks. 
+   * - post_stat_o3
+     - Generates air quality forecast products, including hourly average and statistical products, for O3 (e.g., daily 8-hour average maximum O3). 
+   * - post_stat_pm25
+     - This task generates air quality forecast products, including hourly average and statistical products, for PM2.5 (e.g., 24-hour average PM2.5). 
+   * - bias_correction_o3
+     - Applies a bias-correction technique (e.g., analog ensemble) to improve the raw model forecast for O3 and generates the bias-corrected O3 products. 
+   * - bias_correction_pm25
+     - Applies a bias-correction technique (e.g., analog ensemble) to improve the raw model forecast for PM2.5 and generates the bias-corrected PM2.5 products. 
+
+WE2E Test for AQM
+=======================
+
+Build the app for AQM:
+
+.. code-block:: console
+
+  ./devbuild.sh -p=hera -a=ATMAQ
```

Comment:
It may be worth mentioning here that if the user wants to build AQM from the same clone that they have already built the non-AQM SRW, they can specify a different build and bin directory.

Reply:
@christinaholtNOAA Would they just run `./devbuild.sh -p=hera -a=ATMAQ --bin-dir=NEW_BIN_DIR --build-dir=NEW_BUILD_DIR` from their ufs-srweather-app clone? Is there anything they'll have to do later when configuring an experiment to make sure the experiment can find the right executables?

Reply:
I think they'd need to provide the same (or similar) command line arguments to the test script.

Reply:
@christinaholtNOAA I only see one [test config file for AQM](https://github.com/ufs-community/ufs-srweather-app/blob/develop/tests/WE2E/test_configs/aqm/config.aqm_grid_AQM_NA13km_suite_GFS_v16.yaml), which seems to assume that the app is already built? Can you point me to the test script you're thinking of?